satellite/gracefulexit/endpoint.go

```diff
@@ -1000,3 +957,23 @@ func (endpoint *Endpoint) getValidPointer(ctx context.Context, path string, piec
 	}
 	return pointer, nil
 }
+
+func (endpoint *Endpoint) getNodePiece(ctx context.Context, pointer *pb.Pointer, incomplete *TransferQueueItem) (*pb.RemotePiece, error) {
```

Comment:
nit - we could add a comment clarifying that pointer is expected to be remote when this is called. Alternatively, we could make the `*pb.Pointer` arg a `*pb.RemoteSegment` arg. Not super important, just a thought.